bin/capsule-certs-generate

```diff
@@ -1,10 +1,54 @@
 #!/usr/bin/env ruby
-
 require 'rubygems'
 require 'kafo'
+require 'tempfile'
+
+def self.temp_file(name, content)
+  f = Tempfile.open([name, '.yaml'])
+  f.write content
+  f.close
+  f
+end
+
+INSTALLER_DIR = '.'
+
+ANSWERS_CONTENT = <<EOF
+certs:
+  generate: true
+  deploy: false
+  group: foreman
+  org: Default_Organization
+capsule_certs: true
+EOF
 
-CONFIG_FILE = "./config/capsule-certs-generate.yaml"
+answers_file = temp_file('capsule-certs-answers', ANSWERS_CONTENT)
+
+CONFIG_CONTENT = <<EOF
+:log_level: :debug
+:log_dir: /var/log
+:log_name: capsule-certs-generate.log
+:no_prefix: true
+:answer_file: #{answers_file.path}
+:installer_dir: #{INSTALLER_DIR}
+:module_dirs: ["#{INSTALLER_DIR}/modules"]
```

Comment:
We need to also include the modules from foreman (e.g. /usr/share/foreman-installer/modules); otherwise, running capsules-certs-generate will generate an error similar to:

```
[ERROR 2016-02-09T12:32:21 main]  Unknown function cache_data at /usr/share/foreman-installer-katello/modules/certs/manifests/init.pp:127 on node katello-scenario-katello.example.com
```